src/goto-programs/set_properties.cpp

```diff
@@ -92,7 +101,7 @@ void label_properties(
 void label_properties(goto_programt &goto_program)
 {
   std::map<irep_idt, std::size_t> property_counters;
-  label_properties(goto_program, property_counters);
+  label_properties({}, goto_program, property_counters);
 }
```

Comment:
Can we just remove this function?

Reply:
Done

Reply:
Hmm, that wasn't actually unused, was it? It seems JBMC now fails to build.

Reply:
Now added a variant for JBMC.